src/Aspire.Dashboard/Components/Pages/TraceDetail.razor

```diff
@@ -52,6 +52,13 @@
                     @Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailTotalSpansHeader)] <strong>@trace.Spans.Count</strong>
                 </div>
 
+                <FluentButton Appearance="Appearance.Lightweight" OnClick="@OnCollapseAllAsync" title="Collapse All">
```

Comment:
these buttons should be reading title (the tooltip) from the resources, not hard coding English